models/public/mobilenet-yolo-syg/mobilenet-yolo-syg.md

```diff
@@ -0,0 +1,124 @@
+
+3. The array of detection summary info, name - `separable_conv2d_38/BiasAdd`,  shape - `1,13,13,27`. The anchor values are `142,110,  192,243,  459,401`.
+
+For each case format is `B,Cx,Cy,N*14,`, where
+    - `B` - batch size
+    - `Cx`, `Cy` - cell index
+    - `N` - number of detection boxes for cell
+
+Detection box has format [`x`,`y`,`h`,`w`,`box_score`,`class_no_1`, ..., `class_no_9`], where:
+- (`x`,`y`) - raw coordinates of box center, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get relative to the cell coordinates
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get absolute height and width values
+- `box_score` - confidence of detection box, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get confidence in [0,1] range
+- `class_no_1`,...,`class_no_9` - probability distribution over the classes in logits format, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) 
+								  and multiply by obtained confidence value to get confidence of each class
+
+### Converted model
+
+1. The array of detection summary info, name - `separable_conv2d_22/separable_conv2d/YoloRegion`,  shape - `1,27,52,52`. The anchor values are `12,16,  19,36,  40,28`.
+
+2. The array of detection summary info, name - `separable_conv2d_30/separable_conv2d/YoloRegion`,  shape - `1,27,26,26`. The anchor values are `36,75,  76,55,  72,146`.
+
+3. The array of detection summary info, name - `separable_conv2d_38/separable_conv2d/YoloRegion`,  shape - `1,27,13,13`. The anchor values are `142,110,  192,243,  459,401`.
+
+Detection box has format [`x`,`y`,`h`,`w`,`box_score`,`class_no_1`, ..., `class_no_9`], where:
+- (`x`,`y`) - coordinates of box center relative to the cell
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get absolute height and width values
+- `box_score` - confidence of detection box in [0,1] range
+- `class_no_1`,...,`class_no_9` - probability distribution over the classes in the [0,1] range, multiply by confidence value to get confidence of each class
+
+## Download a Model and Convert it into Inference Engine Format
+You can download models and if necessary convert them into Inference Engine format using the Model Downloader and other automation tools as shown in the examples below.
+An example of using the Model Downloader:
+python3 <omz_dir>/tools/downloader/downloader.py --name <model_name>
+An example of using the Model Converter:
+python3 <omz_dir>/tools/downloader/converter.py --name <model_name>
+
+## Legal Information
+
+The original model is distributed under the following
+[license1](https://raw.githubusercontent.com/tensorflow/models/master/LICENSE):
```

Comment:
what relation is between your original model and tensorflow models garden? As far as I understand you took yolo-v4 model with mobilenet backend from david's repo and retrained it on your data, is not it?